app/controllers/hiring_staff/vacancies_controller.rb

```diff
@@ -30,9 +34,11 @@ def review
     reset_session_vacancy!
     store_vacancy_attributes(@vacancy.attributes)
 
+
     unless @vacancy.valid?
```

Comment:
Maybe this could be a one-liner, the same as on line 24, to keep it consistent

Reply:
There are two lines in the `else` part, so I'm not sure how it could be condensed!

Reply:
(Which is clearer when I get rid of the extra newline in the middle)